### ISSUE TITLE
Ist it maybe an idea to join this with Manifolds.jl?

### ISSUE THREAD

**Issue by kellertuer** [smataigne/StiefelLog.jl#1]
Hi Simon,
Sorry to stumble in here so early after just two commits – and great to see more manifold numerics in Julia!

I could not see much documentation yet on exp and/or log, but it might be this is related to the $\alpha$-metrics by Huper et al?

The have a few cases over in Manifolds.jl also already covered

* Euclidean https://juliamanifolds.github.io/Manifolds.jl/stable/manifolds/stiefel.html#Default-metric:-the-Euclidean-metric
* canonical https://juliamanifolds.github.io/Manifolds.jl/stable/manifolds/stiefel.html#The-canonical-metric
* submersion, which I think they originally called $\alpha$-metric? https://juliamanifolds.github.io/Manifolds.jl/stable/manifolds/stiefel.html#The-submersion-or-normal-metric

Just if that fits well, you could maybe consider contributing these to Manifolds.jl – or if they are available there check whether you could improve our implementation?
I would of course be available to help and happy, if you would contribute this to Manifolds.lj 😎

**Comment by smataigne** [comment]
> d not see much documentation yet on exp and/or log, but it might be this is related to the α-metrics by Huper et al?
> 
> The have a few cases over in Man

Hi Ronny, I would be delighted to contribute to Manifolds.jl. 
This repo is indeed not yet polished, we are first finishing the publication associated. It indeed adresses the computation of the log for the family of  $\alpha$-metrics by Huper et al.

**Comment by kellertuer** [comment]
Nice!

We do have the data structure already for the $\alpha$-metric, but improving its functions is of cause always welcome.
As I said, I am happy to help with technical details, but I also saw, you have some experience with packages already.

And sure, the publication should be finished first – looking forward to reading that as well!
